Resources/config/cache.xml

```diff
@@ -2,15 +2,21 @@
 <container xmlns="http://symfony.com/schema/dic/services"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="smartive_handlebars.cache.redis.client">snc_redis.default</parameter>
+        <parameter key="smartive_handlebars.cache.redis.key_prefix">smartive-handlebars:</parameter>
```

Comment:
this is bad practice. it amounts to using global variables and is completely unchecked. if the user does a typo, or you change this parameter for some reason, it will simply stop working, without triggering any error.

the proper way would be to add a configuration option in the cache section. something like 
```
cache:
    enabled: true
    service: ...
    redis:
         redis_client_service: snc_redis.default
         key_prefix: "smartive-handlebars:"
```

the redis adapter itself is what could go into the handlebars library - the config would stay here so it would make sense to allow proper configuration.

btw, the key prefix configuration is not mentioned in the doc.